src/screens/AirdropDashboard/AirdropDashboardScreen.tsx

```diff
@@ -0,0 +1,104 @@
+import { CompositeNavigationProp } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { FC } from 'react';
+import { View, StyleSheet, Text, TouchableOpacity } from 'react-native';
+
+import { images } from 'app/assets';
+import { Header, ScreenTemplate, Button, Image, Countdown } from 'app/components';
+import { CONST, MainCardStackNavigatorParams, Route, RootStackParams } from 'app/consts';
+import { getFormattedAirdropDate } from 'app/helpers/airdrop';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface Props {
+  navigation: CompositeNavigationProp<
+    StackNavigationProp<RootStackParams, Route.MainCardStackNavigator>,
+    StackNavigationProp<MainCardStackNavigatorParams, Route.AirdropDashboard>
+  >;
+}
+
+export const AirdropDashboardScreen: FC<Props> = ({ navigation }) => (
+  <ScreenTemplate
+    footer={
+      <>
```

Comment:
I would maybe move that out as a separate component to improve readability. Not a necessity though

Reply:
moved to component